### PR TITLE
Convert PNG to SVG (part 7).

### PR DESCRIPTION
**Pull request by JimiC**

### CODE REVIEW

File: src/icon-manifest/supportedExtensions.ts [hunk 1]
```diff
@@ -78,8 +79,8 @@ export const extensions: IFileCollection = {
     { icon: 'cobol', extensions: ['cbl', 'cob', 'cpy'], format: FileFormat.svg },
     { icon: 'codeclimate', extensions: ['codeclimate.yml'], light: true, filename: true, format: FileFormat.svg },
     { icon: 'coffeescript', extensions: [], languages: [languages.coffeescript], format: FileFormat.svg },
-    { icon: 'config', extensions: ['env', 'config'], languages: [languages.properties], format: FileFormat.png },
```
Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: Why did you removed the config extension?

Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: `config` extension is used in `xml` language id.
https://github.com/Microsoft/vscode/blob/1.8.1/extensions/xml/package.json#L16

Comment (JimiC) [inline src/icon-manifest/supportedExtensions.ts]: As filename it's declared in `properties` language id.
https://github.com/Microsoft/vscode/blob/1.8.1/extensions/ini/package.json#L16

Comment (jens1o) [inline src/icon-manifest/supportedExtensions.ts]: Okey, thanks for the clarification.

**Pull request merged**